SPEC.md

```diff
@@ -257,7 +257,8 @@ The list is described in JSON form, and can be stored on disk or generated from
 - `plugins` (list): A list of standard CNI network configuration dictionaries (see above).
 
 When executing a plugin list, the runtime MUST replace the `name` and `cniVersion` fields in each individual network configuration in the list with the `name` and `cniVersion` field of the list itself.
-This ensures that the name and CNI version is the same for all plugin executions in the list, preventing versioning conflicts between plugins.
+The runtime MUST also send the same `runtimeConfig` field to each plugin in the list, if any `runtimeConfig` field is sent.
```

Comment:
I'm not sure about this point - why are you passing the same runtime config to all plugins? This goes directly against what I had in mind with the conventions stuff.